apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -68,12 +190,38 @@
 
   li {
     display: inline-block;
-    width: 100px;
-    margin: 0 10px;
+    width: 49%;
+    height: 0;
+    margin: 0;
     cursor: pointer;
+    padding-bottom: 50%;
+    background-repeat: no-repeat;
+    background-size: 85% 85%;
+    background-position: center center;
+    background-color: white;
+    margin-bottom: 2%;
 
-    img {
-      width: 100%;
+    &:nth-child(odd) {
+      margin-right: 2%;
+    }
+
+    &.selected {
+      background-color: #999;
     }
   }
 }
+
+.reaction-button {
+  background-color: #2f7791;
+  cursor: pointer;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  padding: 4%;
+
+  & > svg {
```

Comment:
Unnecessary parent selector (&)

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#unnecessaryparentreference)